SimpleColorApp/SimpleColorApp/ContentView.swift

```diff
@@ -8,17 +8,56 @@
 import SwiftUI
 
 struct ContentView: View {
-    var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundStyle(.tint)
-            Text("Hello, world!")
-        }
-        .padding()
+  
+  @State private var redSliderValue: Double
+  @State private var greenSliderValue: Double
+  @State private var blueSliderValue: Double
+  @State private var boxColor:Color
+  
+  init() {
```

Comment:
Solving the problem of having default values and then changing them is the tricky part of this and most of the solutions you know about at this stage are going to be messy. Another way to solve this would have been to inject the defaults like
````
init(red: Double = 200.0, green: Double = 100.0, blue: Double = 240.0) {
    self.redSliderValue = red
    self.greenSliderValue = green
    self.blueSliderValue = blue
    self.boxColor = ContentView.calculateBoxColor(red: red, green: green, blue: blue)
  }
````
Just trying to point out some alternative ways this can get handled. Your way works, and meets the requirements, so it is also correct.